tools/Builder/config.json

```diff
@@ -159,9 +159,15 @@
         },
         {
             "folder": "Vindicta.isladuala3",
-            "CfgMissionsEntry": "vindicta_isla_duala",
```

Comment:
Hmm so I guess it was broken, right? Or which one is wrong?

Reply:
No, that was an accident on my part.